docs/components/label.md

```diff
@@ -77,18 +77,41 @@ Use the `for` attribute to bind a label to an input element for accessibility.
 <KInput id="service"/>
 ```
 
+## Slots
+
+- `tooltip` - Rather than using the `help` or `info` props, if you want to style the tooltip content you can use this slot.
+
+:::tip Note:
+By default, when using the `tooltip` slot the `info` `KIcon` will be shown, but you can get the `help` icon to display instead by setting the `help` prop to any non-empty string value.
+:::
+
+<KLabel help="true">
+  My Tooltip
+  <template #tooltip>Brings all the <code>devs</code> to the yard</template>
+</KLabel>
+<KInput />
+
+```html
+<KLabel help="true">
```

Comment:
nit: can we remove the `help` attribute in the code and example since we're using the slot?

Reply:
It's there because this example is showing how you can get the `help` icon (instead of `info`) when using the slot.

Reply:
Ah that makes sense. Maybe add a note about how to specify the icon in both of the docs pages?